### PR TITLE
[IMP] open_academy: Custom course views T#59081

### PR DESCRIPTION
**Pull request by JesusValdez96**
- Rename course_menu.xml to course_views.xml due to added views.
- Add Tree view as main view.
- Add Form view to put the description field under a tab.
- Add Search view to allow searching based on title and description.

Link to exercise: https://www.odoo.com/documentation/15.0/developer/howtos/backend.html#form-views

### CODE REVIEW

**Comment by JesusValdez96** [comment]
Please review @CarmenMiranda, @andreaarce

File: open_academy/views/course_views.xml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        </field>
+    </record>
+
+    <record id="course_view_form" model="ir.ui.view">
+        <field name="name">course.view.form</field>
+        <field name="model">course</field>
+        <field name="arch" type="xml">
+            <form string="Course form">
+                <field name="title"/>
+
```
Comment (CarmenMiranda) [inline open_academy/views/course_views.xml:28]: Can we avoid this empty line? 😄

**Review by CarmenMiranda — CHANGES_REQUESTED** [review]
Just a little comment, in the commit you said you are changing the form view, but in reality, you are adding it

**Comment by JesusValdez96** [comment]
Changes made, Thanks @CarmenMiranda, please review again. :smile:

**Review by CarmenMiranda — APPROVED** [review]
@andreaarce Technically LGTM 👍🏼 

@JesusValdez96 Just remove the Draft of the PR 🙈

**Pull request merged**